src/app/app.module.ts

```diff
@@ -10,9 +10,10 @@ import { AppRoutingModule } from './app-routing.module';
 import { HomeComponent } from './components/home/home.component';
 import { PageNotFoundComponent } from './components/layout/page-not-found/page-not-found.component';
 import { HeaderComponent } from './components/layout/header/header.component';
-import { SearchUserComponent } from './components/search-user/search-user.component';
+import { UserSearchComponent } from './components/user-search/user-search.component';
```

Comment:
We can't name this as `UserSearchComponent` as its going to have more than just search functionality in future. The names need to generic enough to handle future scenarios. I would prefer this to be named `UserContainerComponent`.